src/runpb.c

```diff
@@ -228,7 +228,11 @@ read_runpb_ic(FastPMSolver * fastpm, FastPMStore * p, const char * filename)
     }
 
     const double omega = OmegaA(aa, fastpm->cosmology);
-    const float DplusIC = GrowthFactor(aa, fastpm->cosmology);
+
+    //FIXME: Make object just for D1?
+    FastPMGrowthInfo gi;
+    fastpm_growth_info_init(&gi, aa, fastpm->cosmology);
```

Comment:
Since we always need to solve the ODE, an object just for D1 is probably not going to be a lot faster; and we only call this at IC generation.

The space behind comment '//' -- Here is a reference on how weird it looks without the space:
https://stackoverflow.com/a/1496218

Reply:
Yeah, I agree, it's just a bad habit when I know the comment will be temporary, but I'll stop doing this!